packages/react-component-library/cypress/specs/DatePicker/index.spec.ts

```diff
@@ -39,25 +35,13 @@ describe('DatePicker', () => {
         cy.get(selectors.datePicker.input).should('have.value', expected)
       })
 
-      it('should not be in an error state', { browser: '!firefox' }, () => {
+      it('should not be in an error state', () => {
         cy.get(selectors.datePicker.outerWrapper).should(
-          'have.css',
-          'border',
-          `1px solid ${hexToRgb(ColorNeutral200)}`
+          'has.border',
+          'color',
+          hexToRgb(ColorNeutral200)
```

Comment:
I've only done assertions on the border colour as that seems like the relevant thing here.